File: Programming Basics with Python/06 Nested loops/More exercises/12_The_song_of_the_wheels.py
control_number = int(input())
counter = 0
password = "No!"
for a in range(1, 10):
    for b in range(1, 10):
        for c in range(1, 10):
            for d in range(1, 10):
                if a < b and c > d and control_number == a * b + c * d:
                    print(f"{a}{b}{c}{d}", end=" ")
                    counter += 1
                    if counter == 4:
                        password = f"Password: {a}{b}{c}{d}"
print()
print(password)

File: JS Basics/06 Nested Loops - More Exercises/12. The song of the wheels.js
function song(input) {
    let controlNumber = Number(input[0]);
    let counter = 0;
    let pass = 0;
    let number = "";
    for (let numberOne = 1; numberOne <= 9; numberOne++) {
        for (let numberTwo = 1; numberTwo <= 9; numberTwo++) {
            for (let numberThree = 1; numberThree <= 9; numberThree++) {
                for (let numberFour = 1; numberFour <= 9; numberFour++) {
                    if ((numberOne < numberTwo) && (numberThree > numberFour)) {
                        if (numberOne * numberTwo + numberThree * numberFour == controlNumber) {
                            number += (`${numberOne}${numberTwo}${numberThree}${numberFour} `);
                            counter++;
                            if (counter == 4) {
                                pass = `${numberOne}${numberTwo}${numberThree}${numberFour}`;
                            }
                        }
                    }
                }
            }
        }
    }
    console.log(number);
    if (counter >= 4) {
        console.log(`Password: ${pass}`);
    }
    else {
        console.log("No!");
    }
}
song(["11"])